test/clt-tests/replication/replication-test.rec

```diff
@@ -1,189 +1,689 @@
-content: Macbook
-gid: 3
+model: Google Pixel 6
+storage_capacity: 128
+color: sorta seafoam
+release_year: 2021
+price: 699.989990
+discounted_price: 599.989990
+sold: 1
+date_added: 3230359520
+product_codes: 33,34,35
+values: 623456764345678976,9223372036854775807
+additional_info: {"features":["Google Tensor chip","Flat OLED display","Dual-camera system"]}
+vector: 0.823448,0.362478,0.187971,0.509821
 *************************** 4. row ***************************
 id: 6775836565403860996
-title: NEW
-content: TABLE
-gid: 4
+model: OnePlus 9
+storage_capacity: 256
+color: winter mist
+release_year: 2021
+price: 729.989990
+discounted_price: 629.989990
+sold: 0
+date_added: 3072057632
+product_codes: 36,37,38
+values: 723456764345678976,9223372036854775807
+additional_info: {"features":["Qualcomm Snapdragon 888","Fluid AMOLED display","Hasselblad Camera for Mobile"]}
+vector: 0.021106,0.918278,0.261892,0.074594
+––– input –––
+sleep 5;mysql -h0 -P2306 -e "SELECT * FROM testrt ORDER BY id ASC\G"
+––– output –––
+*************************** 1. row ***************************
+id: 6775836565403860993
+model: iPhone 13 Pro
+storage_capacity: 256
+color: silver
+release_year: 2021
+price: 1099.989990
+discounted_price: 989.989990
+sold: 1
+date_added: 2224442480
+product_codes: 1,2,3
+values: 523456764345678976,9223372036854775807
+additional_info: {"features":["ProMotion display","A15 Bionic chip","Ceramic Shield front cover"]}
+vector: 0.773448,0.312478,0.137971,0.459821
+*************************** 2. row ***************************
+id: 6775836565403860994
+model: Galaxy S21 Ultra
+storage_capacity: 128
+color: black
+release_year: 2021
+price: 1199.989990
+discounted_price: 1099.989990
+sold: 0
+date_added: 3141431296
+product_codes: 4,5,6
+values: 1234567890123456789,9223372036854775807
+additional_info: {"features":["Dynamic AMOLED 2X","Exynos 2100","108MP camera"]}
+vector: 0.500000,0.400000,0.300000,0.200000
+*************************** 3. row ***************************
+id: 6775836565403860995
+model: Google Pixel 6
+storage_capacity: 128
+color: sorta seafoam
+release_year: 2021
+price: 699.989990
+discounted_price: 599.989990
+sold: 1
+date_added: 3230359520
+product_codes: 33,34,35
+values: 623456764345678976,9223372036854775807
+additional_info: {"features":["Google Tensor chip","Flat OLED display","Dual-camera system"]}
+vector: 0.823448,0.362478,0.187971,0.509821
+*************************** 4. row ***************************
+id: 6775836565403860996
+model: OnePlus 9
+storage_capacity: 256
+color: winter mist
+release_year: 2021
+price: 729.989990
+discounted_price: 629.989990
+sold: 0
+date_added: 3072057632
+product_codes: 36,37,38
+values: 723456764345678976,9223372036854775807
+additional_info: {"features":["Qualcomm Snapdragon 888","Fluid AMOLED display","Hasselblad Camera for Mobile"]}
+vector: 0.021106,0.918278,0.261892,0.074594
+––– input –––
+sleep 5;mysql -h0 -P2306 -e "SELECT * FROM testrt ORDER BY id ASC\G"
```

Comment:
What is this sleep for?

Reply:
Why did you call `mysql -h0 -P2306 -e "SELECT * FROM testrt ORDER BY id ASC\G"` 3 times?